Updates README to desribe recently added features (#29)
antmicro/renode-linux-runner-action: README.md

## Parameters

> - `shared-dir` - Path to the shared directory. Contents of this directory will be mounted in Renode. The embedded Linux in Renode will start in this directory. Any other file from your repository/docker container will not be visible.

Reviewer: ```suggestion
- `shared-dir` - Path to the shared directory. Contents of this directory will be mounted in Renode. The embedded Linux in Renode will start in this directory. Any other file from your repository/Docker container will not be visible.
```

Multiple devices can also be specified this way.

The [release](.github/workflows/release.yml) workflow contains an example usage of this action.

### Python packages

> This action offers sideloading your Python packages that you want to use in emulated Linux. You can select any package from the pypi repository
> or from the git repository.

Reviewer: ```suggestion
This action offers sideloading Python packages that you want to use in the emulated system. You can select any package from PyPI or from a Git repository.
```

For example:

```yaml
- uses: antmicro/renode-linux-runner-action@v0
  with:
    shared-dir: ./shared-dir
    renode-run: python --version
    python-packages: |
      git+https://github.com/antmicro/pyrav4l2.git
      pytest
```

You can also pass the specific version requirement:

```yaml
- uses: antmicro/renode-linux-runner-action@v0
  with:
    shared-dir: ./shared-dir
    renode-run: python --version
    python-packages: |
      git+https://github.com/antmicro/pyrav4l2.git@3c071a7
      pytest==5.3.0
      pyyaml>=5.3.1
```

The action will download all selected packages and their dependencies and install them later in the emulated Linux environment.

## Git repositories

> If you want to use some files that are outside of your repository, you can sideload them to the emulated Linux with the `repos` argument:

Reviewer: ```suggestion
If you want to clone other Git repositories to the emulated system, you can use the `repos` argument:
```

```yaml
- uses: antmicro/renode-linux-runner-action@v0
  with:
    shared-dir: ./shared-dir
    renode-run: python --version
    repos: https://github.com/antmicro/pyrav4l2.git
```

> You can also specify the catalog name into which you want to clone the repository:

Reviewer: ```suggestion
You can also specify the directory into which you want to clone the repository:
```
Can it be a path with slashes? If so, maybe use _path_ instead of _directory_.
